research-app-messages/src/classic_pywwt.ts

```diff
@@ -209,6 +212,7 @@ export function isCreateImageSetLayerMessage(o: any): o is CreateImageSetLayerMe
     o.event == "image_layer_create" &&
     typeof o.id === "string" &&
     typeof o.url === "string" &&
+    typeof o.mode  === "string" &&
     (o.mode == "autodetect" || o.mode == "fits" || o.mode == "preloaded");
```

Comment:
I'm pretty sure that it's a bit overkill, but I think the type guard function here should validate that `goto` is either undefined or a boolean. Since all JS objects have "truthiness" other types can work, but I think it is better to err on the side of being rigorous about input checking.

Reply:
I had it, but lost it :disappointed: https://github.com/WorldWideTelescope/wwt-webgl-engine/pull/97/commits/7eb7edb7a2961e4ff82a682fb838772873c12636